app/models/presenters/rvi_cdna_xp_presenter.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Presenters
+  # Used for RVI cDNA XP plates. Used for a custom 'Manual Transfer' state transition.
+  # N.B. 'Manual Transfer' is used for the RVI BCL pipeline for the cDNA XP Plate because the plates
+  # are inaccessible in the bed at the time of transfer so cannot be transferred via
+  # bed verifications. Also we cannot set its state to 'Passed' on initial bed verification
+  # because there are steps on other plates that need to be completed before the cDNA XP Plate can be transferred.
+  # and we do not want to misreport the plate being passed if an issue arises with the steps before hand.
+  class RviCdnaXpPresenter < PlatePresenter
```

Comment:
Minor thing but the capitalisation in the classname looks odd to me.
`RVIcDNAXpPresenter`?
RVI and DNA are already initialisms of longer titles.
c in cDNA means complimentary and is written in lower case.
Dunno if doing that causes any class loading issues mind.

Reply:
I agree, not the nicest capitalisation, but having the name as anything else causes loading issues unfortunately.